src/server/user.ts

```diff
@@ -0,0 +1,17 @@
+
+import { User, UserSql } from '../types/User';
+
+export async function fetchUserDetails(): Promise<User> {
+    const user: UserSql[] = await fetch('/user').then(response => {
```

Comment:
Perhaps return a `user[0]`, since you fetch only 1 (Expect: `const user: UserSql[]`)

Reply:
not in used anymore, old code